src/Symfony/Component/Notifier/Bridge/AmazonSns/composer.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/amazon-sns-notifier",
+    "type": "symfony-bridge",
+    "description": "Provides Amazon SNS integration for Symfony Notifier.",
+    "keywords": ["amazon", "sns", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Adrien Chinour",
+            "email": "github@chinour.fr"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.3|^5.0",
+        "symfony/notifier": "^5.3",
+        "async-aws/sns": "^1.0",
+        "async-aws/core": "^1.7"
```

Why not bump these to the latest version?

```suggestion
        "async-aws/sns": "^1.2",
        "async-aws/core": "^1.9"
```